x-pack/plugins/security_solution/public/management/pages/trusted_apps/store/selectors.ts

```diff
@@ -74,6 +74,18 @@ export const getCurrentLocationFilter = (state: Immutable<TrustedAppsListPageSta
   return state.location.filter;
 };
 
+export const getCurrentLocationIncludedPolicies = (
```

Comment:
Q. don't you need to use these as well in the `needsRefreshOfListData()` selector above? in order to ensure we aer not doing un-necessary api calls?

Reply:
Not at all, I'm using the `handleOnSearch` method in  `trusted_apps_page` when there is some change in that component that needs to trigger a search (click on refresh button, submit search input or change some policy) so no need to add policies here as it would call twice.